### ISSUE TITLE
More explanation of modulo

### ISSUE THREAD

**Issue by maneesha** [datacarpentry/python-ecology-lesson#424]
The episode [Short Introduction to Programming in Python](https://datacarpentry.org/python-ecology-lesson/01-short-introduction-to-Python/index.html) notes that 

> We can perform mathematical calculations in Python using the basic operators +, -, /, *, %:

I don't think most people would consider `modulo` to be a basic operator.  I would recommend some more explanation of what modulo does.

**Comment by maxim-belkin** [comment]
> I don't think most people would consider `modulo` to be a basic operator.

I agree. Would dropping it here be a reasonable alternative to explaining it?

**Comment by maneesha** [comment]
Either way would work from my perspective.  It could be useful to know, so it may be worth introducing.  However, that may make this lesson longer so I'm fine dropping it as well.

**Comment by nettic** [comment]
There is a good example which shows either both of things which I miss here: the if/else statement and the example for modulo:

There is one common use for the Modulo Operator: to find even or odd numbers eg in a range 1 to 10: 
for number in range(1, 10):
    if(number % 2 != 0):
        print(number)

The lesson shows the boolean type but no example for the usage. If/else condition would be helpful to introduce at this point. Thus, both topics are great to include here.

**Comment by gnguyen27** [comment]
I would suggest to add the example of finding even or odd numbers as an exercise to the section when function is introduced.

[https://datacarpentry.org/python-ecology-lesson/01-short-introduction-to-Python/#functions](url)

Provide an example of printing out even number between 1 and 10.

```
for number in range(1, 10):
        if (number %2 != 0) :
            print(number)
```

Exercise:
* Modify the code above to print out odd numbers
* Based on the example code above , write a function find_even that takes two inputs and print out even number
